### PR TITLE
feature(export) shopware6 multimedia service and customfield improvement

### PR DESCRIPTION
**Pull request by wfajczyk**
# Description

Add multimedia to export and improvement in custom field. 

Issue #528
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

- [x] Unit Tests
- [ ] Behat Test

# Checklist:
- [x] I have read the contribution requirements and fulfill them.
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: module/exporter-shopware-6/src/Infrastructure/Connector/SwagQLBuilder.php [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types = 1);
+
+namespace Ergonode\ExporterShopware6\Infrastructure\Connector;
+
+/**
+ */
+class SwagQLBuilder
```
Comment (rprzedzik) [inline module/exporter-shopware-6/src/Infrastructure/Connector/SwagQLBuilder.php:13]: what is SwagQLBuilder ?

File: module/exporter-shopware-6/src/Infrastructure/Synchronizer/CategorySynchronizer.php [hunk 2]
```diff
@@ -222,7 +217,9 @@ private function getShopwareCategory(Shopware6Channel $channel, string $name): ?
      */
     private function getShopwareCategoryList(Shopware6Channel $channel): array
     {
-        $action = new GetCategoryList();
+        $query = new SwagQLBuilder();
+        $query->limit(500);
```
Comment (rprzedzik) [inline module/exporter-shopware-6/src/Infrastructure/Synchronizer/CategorySynchronizer.php:221]: What happen if there is more then 500 categories ??

File: module/exporter-shopware-6/src/Infrastructure/Connector/SwagQLBuilder.php (hunk 1 above)
Comment (wfajczyk) [reply module/exporter-shopware-6/src/Infrastructure/Connector/SwagQLBuilder.php]: Builder to query in Shopware API. I changed to `Shopware6QueryBuilder`

File: module/exporter-shopware-6/src/Infrastructure/Synchronizer/CategorySynchronizer.php (hunk 2 above)
Comment (wfajczyk) [reply module/exporter-shopware-6/src/Infrastructure/Synchronizer/CategorySynchronizer.php:221]: Because, default API value is 10 or 20

**Pull request merged**